source/guides/administration.rst

```diff
@@ -28,8 +29,9 @@ The Basics
     Multi-factor Authentication </onboard/multi-factor-authentication>
     Active Directory / LDAP </onboard/ad-ldap>
 
-These resources will help you get started with your Mattermost server.
+These resources will help you get started with your Mattermost workspace.
 
+* Visit the :doc:`Workspace Optimization </configure/optimize-your-workspace>` page in the System Console to review health and growth scores for your Mattermost workspace and take recommended actions.
```

Comment:
Curious as to why this entry's format is different to all the others? I'm all for it, it reads nicely. :)

Reply:
Are you referring to the relative link syntax?